README.md

```diff
@@ -177,7 +193,7 @@ try {
         String fullName = profile.GetFullName();
         String givenNames = profile.getGivenNames();
         String familyName = profile.getFamilyName();
-        String mobileNumber = profile.getPhoneNumber();
+        String phoneNumber = profile.getPhoneNumber();
```

Comment:
This snippet should use the `Attribute` class now

Reply:
Good spot, thanks.